docs/tutorials/tutorial-hdfs-logs.md

```diff
@@ -30,76 +29,62 @@ Let's download and install Quickwit.
+for the nested values `resource.service` . It also sets the `default_search_fields`, the `tag_fields`, and the `timestamp_field`.The `timestamp_field` and `tag_fields` are used by Quickwit for [splits pruning](../overview/architecture.md) at query time to boost search speed. Check out the [index config docs](../reference/index-config.md) for more details.
+
+```yaml title="hdfslogs_index_config.yaml"
+version: 0
+
+doc_mapping:
+  field_mappings:
+    - name: timestamp
+      type: i64
+      fast: true # Fast field must be present when this is the timestamp field.
+    - name: severity_text
+      type: text
+      tokenizer: raw # No tokeninization.
+    - name: body
+      type: text
+      tokenizer: default
+      record: position # Record position will enable phrase query on body field.
+    - name: resource
+      type: object
+      field_mappings:
+        - name: service
+          type: text
+          tokenizer: raw # Text field referenced as tag must have the `raw` tokenier.
+  tag_fields: [resource.service]
+  store_source: true
+
+indexing_settings:
+  timestamp_field: timestamp
+
+search_settings:
+  default_search_fields: [severity_text, body]
+```
 
+Now let's create the index with the `create` subcommand (assuming you are inside quickwit install directory):
 
-```json title="hdfslogs_index_config.json"
-{
-    "store_source": false, // The document source (=json) will not be stored.
-    "default_search_fields": ["body", "severity_text"],
-    "timestamp_field": "timestamp",
-    "field_mappings": [
-        {
-            "name": "timestamp",
-            "type": "i64",
-            "fast": true // Fast field must be present when this is the timestamp field.
-        },
-        {
-            "name": "severity_text",
-            "type": "text",
-            "tokenizer": "raw" // No tokeninization.
-        },
-        {
-            "name": "body",
-            "type": "text",
-            "tokenizer": "default",
-            "record": "position" // Record position will enable phrase query on body field.
-        },
-        {
-            "name": "resource",
-            "type": "object",
-            "field_mappings": [
-                {
-                    "name": "service",
-                    "type": "text",
-                    "tokenizer": "raw"
-                }
-            ]
-        },
-        {
-            "name": "attributes",
-            "type": "object",
-            "field_mappings": [
-                {
-                    "name": "class",
-                    "type": "text",
-                    "tokenizer": "raw"
-                }
-            ]
-        }
-    ]
-}
-```
+Let's make use of the default configuration file provided in quickwit installation throughout this guide by setting the `QW_CONFIG` environment variable.
 
-Now we can create the index with the new command (assuming you create it in your current directory):
+```bash
+export QW_CONFIG=./config/quickwit.yaml
```

Comment:
Add a small comment just to say that all commands need quickwit config and it's a good practice to have it in this env variable.